diff --git a/node_modules/@midwayjs/serverless-app/dist/configuration.js b/node_modules/@midwayjs/serverless-app/dist/configuration.js
index 2a431cf..8dcd3b7 100644
--- a/node_modules/@midwayjs/serverless-app/dist/configuration.js
+++ b/node_modules/@midwayjs/serverless-app/dist/configuration.js
@@ -15,7 +15,6 @@ const FaaS = require("@midwayjs/faas");
 const framework_1 = require("./framework");
 let ServerlessAppConfiguration = class ServerlessAppConfiguration {
     async onServerReady() {
-        await this.framework.run();
     }
 };
 __decorate([
diff --git a/node_modules/@midwayjs/serverless-app/dist/utils.js b/node_modules/@midwayjs/serverless-app/dist/utils.js
index 5e46209..aef81e2 100644
--- a/node_modules/@midwayjs/serverless-app/dist/utils.js
+++ b/node_modules/@midwayjs/serverless-app/dist/utils.js
@@ -112,6 +112,20 @@ const analysisDecorator = async (cwd) => {
             }
         });
     }
+
+    if (Array.isArray(global['HOOKS_ROUTER'])) {
+        for(const router of global['HOOKS_ROUTER']) {
+          allFunc[router.functionId] = {
+            handler: router.handler,
+            events: [
+              {
+                [router.type]: router
+              }
+            ]
+          }
+        }
+      }
+
     return allFunc;
 };
 exports.analysisDecorator = analysisDecorator;
