diff --git a/node_modules/@midwayjs/koa/dist/framework.js b/node_modules/@midwayjs/koa/dist/framework.js
index 1ed0d74..8b22eab 100644
--- a/node_modules/@midwayjs/koa/dist/framework.js
+++ b/node_modules/@midwayjs/koa/dist/framework.js
@@ -118,7 +118,7 @@ let MidwayKoaFramework = class MidwayKoaFramework extends core_1.BaseFramework {
         const customPort = (_a = process.env.MIDWAY_HTTP_PORT) !== null && _a !== void 0 ? _a : this.configurationOptions.port;
         if (customPort) {
             new Promise(resolve => {
-                const args = [this.configurationOptions.port];
+                const args = [customPort];
                 if (this.configurationOptions.hostname) {
                     args.push(this.configurationOptions.hostname);
                 }
