diff --git a/node_modules/@midwayjs/fcli-plugin-package/dist/index.js b/node_modules/@midwayjs/fcli-plugin-package/dist/index.js
index 1331a4a..2f4fcc0 100644
--- a/node_modules/@midwayjs/fcli-plugin-package/dist/index.js
+++ b/node_modules/@midwayjs/fcli-plugin-package/dist/index.js
@@ -179,7 +179,8 @@ class PackagePlugin extends command_core_1.BasePlugin {
             (0, path_1.join)(cwd, 'midway.config.js'),
         ].find(file => (0, fs_extra_1.existsSync)(file));
         if (midwayConfig) {
-            const modInfo = (0, command_core_2.findNpmModule)(cwd, '@midwayjs/hooks-core');
+            const modInfo = (0, command_core_2.findNpmModule)(cwd, '@midwayjs/hooks/internal') ||
+                (0, command_core_2.findNpmModule)(cwd, '@midwayjs/hooks-core');
             if (modInfo) {
                 const { getConfig } = require(modInfo);
                 const config = getConfig(cwd);
