diff --git a/node_modules/@preconstruct/cli/cli/dist/cli.cjs.dev.js b/node_modules/@preconstruct/cli/cli/dist/cli.cjs.dev.js
index 5e93a2f..cc0fa1a 100644
--- a/node_modules/@preconstruct/cli/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@preconstruct/cli/cli/dist/cli.cjs.dev.js
@@ -1859,6 +1859,10 @@ let getRollupConfig = (pkg, entrypoints, aliases, type, reportTransformedFile) =
     external.push(...Object.keys(pkg.json.peerDependencies));
   }
 
+  if (pkg.json.devDependencies) {
+    external.push(...Object.keys(pkg.json.devDependencies));
+  }
+
   if (pkg.json.dependencies && type !== "umd") {
     external.push(...Object.keys(pkg.json.dependencies));
   }
